pyipn/detector.py

```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import astropy.constants as constants
-
+import re
 
```

Comment:
regex can be very slow as it compiles each time. In python it is easier for simple searches like contains to just say 

```python 
if "GALILEO" in self._name
```
as it is super fast.